docs/dal/index.rst

```diff
@@ -146,10 +146,168 @@ Table Access Protocol
 
     -- `Table Access Protocol <https://www.ivoa.net/documents/TAP/>`_
 
+Getting started
+^^^^^^^^^^^^^^^
+
+Consider the following example for using TAP and ADQL, retrieving 5
+objects from the GAIA DR3 database, showing their id, position and
+mean G-band magnitude between 19 - 20:
+
 .. doctest-remote-data::
 
+    >>> import pyvo as vo
     >>> tap_service = vo.dal.TAPService("http://dc.g-vo.org/tap")
-    >>> tap_results = tap_service.search("SELECT TOP 10 * FROM ivoa.obscore")
+    >>> tap_service.search(
+    >>>        """ SELECT TOP 5 
+    >>>        source_id, ra, dec, phot_g_mean_mag 
+    >>>        FROM gaia.dr3lite
+    >>>        WHERE phot_g_mean_mag BETWEEN 19 AND 20
+    >>>        ORDER BY phot_g_mean_mag
+    >>>        """)
+    <Table length=5>
+        source_id              ra                dec         phot_g_mean_mag
+                            deg                deg               mag      
+        int64             float64            float64           float32    
+    ------------------- ------------------ ------------------ ---------------
+    2162809607452221440 315.96596187101636 45.945474015208106            19.0
+    2000273643933171456  337.1829026565382   50.7218533537033            19.0
+    2171530448339798784  323.9151025188806  51.27690705826792            19.0
+    2171810342771336704 323.25913736080776  51.94305655940998            19.0
+    2180349528028140800  310.5233961869657   50.3486391034819            19.0
+
+If you know a TAP service's access URL, you can directly pass it to
+:py:class:`~pyvo.dal.TAPService` to obtain a service object. 
+Sometimes, such URLs are published in papers or passed around through
+other channels. Most commonly, you will discover them in the VO 
+registry (cf. :ref:`pyvo.registry<pyvo-registry>`).
+
+To perform a query using ADQL, the ``search()`` method is used. 
+TAPService instances have several methods to inspect the metadata
+of the service - in particular, what tables with what columns are
+available - discussed below.  Note that for exploratory query
+construction, you can also use interactive TAP clients such as
+TOPCAT_, which include table browsers.
+
+.. _TOPCAT: http://www.star.bris.ac.uk/~mbt/topcat/ 
+
+To get an idea of how to write queries in ADQL, have a look at
+`GAVO's ADQL course`_; it is basically a standardised subset of SQL
+with some extensions to make it work better for astronomy.
```

Comment:
The definition of ADQL is already provided in the first paragraph, no need for the second half of the sentence. 
I'm also on the fence about the ADQL tutorial, but if @msdemlei wants to have the link here, then it should be fine.

Reply:
Well, I give you the ADQL course shows its age, but frankly I think it would be a good idea to link to *some* accessible ADQL resource.  The ADQL spec definitely is a terrible place to send anyone but an implementor to...

So, I'm happy to yield to any other gentle ADQL intro, but we really should avoid sending folks new to SQL-like stuff to IVOA documentation, and not saying anything about "what's this ADQL that I'm supposed to enter" isn't a good option either.

Talking about which: I think we're not saying anything about TAP examples anywhere yet?  They're also great to get people over the first few metres (try the examples attribute of a TAPService object).  Would this fit here, in addition?